modules/cluster-logging-release-notes-5.1.0.adoc

```diff
@@ -80,5 +80,5 @@ If you configure the `ClusterLogging` custom resource (CR) to switch from epheme
 +
 (link:https://issues.redhat.com/browse/LOG-1351[*LOG-1351*])
 
-Before this update, if you redeployed a full Elasticsearch cluster, it got stuck in an unhealthy state, with one non-data node running and all other data nodes shut down. This happened because new certificates prevented the Elasticsearch Operator from scaling down the non-data nodes of the Elasticsearch cluster. With this update, Elasticsearch Operator can scale all the data and non-data nodes down and then back up again, so they load the new certificates. The Elasticsearch Operator can reach the new nodes after they load the new certificates.
+* Before this update, if you redeployed a full Elasticsearch cluster, it got stuck in an unhealthy state, with one non-data node running and all other data nodes shut down. This happened because new certificates prevented the Elasticsearch Operator from scaling down the non-data nodes of the Elasticsearch cluster. With this update, Elasticsearch Operator can scale all the data and non-data nodes down and then back up again, so they load the new certificates. The Elasticsearch Operator can reach the new nodes after they load the new certificates.
```

Comment:
the phrasing `it got stuck in an unhealthy state` sounds a little like a colloquialism. 

Could you rephrase to something like, 

`Before this update, if you redeployed a full Elasticsearch cluster, it would have one non-data node running and all other data nodes shut down. `

Reply:
I wondered the same thing, Jake, and checked the documentation. The term, "unhealthy," appears in the UI and many parts of our documentation.